app/views/orders/show.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="col-sm-12 col-sm-offset-2">
+  <table class="table table-hover table-stripped table-responsive">
+    <thead>
+      <tr>
+        <td colspan='2'>Buku</td>
+        <td>Harga Satuan</td>
+        <td>Kuantitas</td>
+      </tr>
+    </thead>
+    <% @order.order_details.each do |order_detail|%>
+      <tr>
+        <td>
+          <img src='https://images-na.ssl-images-amazon.com/images/I/51HWaRzG42L._SS100_.jpg' alt='iPhone'>
+        </td>
+        <td>
+          <%= order_detail.book.title %>
+        </td>
+
+        <td>
+          <%= money_amount(order_detail.unit_price) %>
+        </td>
+
+        <td>
+          <div class="inline-block">
+            <%= form_for order_detail, url: order_order_detail_path(order_id: @order.id, id: order_detail.id) do |f| %>
```

Comment:
It is not, but the book wouldn't be able to contain it (> 80 characters) properly, because of the word wrap.

In my personal capacity as reader, I don't like it when opening a programming book and the line is wrapped to the next line, it makes it hard to read.